targets/Targets.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <LangVersion>latest</LangVersion>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>netcoreapp2.0</TargetFramework>

```

Comment:
Consider `netcoreapp2.1`?

Reply:
Can do. I just went with a version that was supported by my current SDK. Is 2.0 going to be a problem?

Reply:
2.0 would be fine. I just thought: why not latest and greatest?